canonical-web-and-design/vanilla-framework
# Better dotrun support

## Done

- adds dotrun generated files to gitignore
- adds `start` script to package.json

## QA

- Pull code
- If you have dotrun snap installed, run `dotrun`
- Otherwise `./run exec yarn start`
- Vanilla should be built, server should start
